backtesting/_plotting.py

```diff
@@ -207,6 +216,16 @@ def plot(*, results: pd.Series,
         filename = _windos_safe_filename(str(results._strategy))
     _bokeh_reset(filename)
 
+    # Calculate proportional heights based on original ratios
+    # Original OHLC height was 400px, use that as base ratio
+    base_height = plot_height
+    equity_height = int(base_height * 0.25)  # was 100px
+    equity_height_with_dd = int(base_height * 0.2)  # was 80px
+    drawdown_height = int(base_height * 0.2)  # was 80px
+    pl_height = int(base_height * 0.2)  # was 80px
+    volume_height = int(base_height * 0.175)  # was 70px
+    indicator_height = int(base_height * 0.125)  # was 50px
```

Comment:
I don't like that these are fixed proportions, and that's that.

Maybe we could make param `plot_height: int | Tuple[int] | Dict[str, int]`? :thinking: